examples/src/status-deserialize-class.js

```diff
@@ -28,7 +28,8 @@ class Truck {
     static schema = {
         name: "string",
         speed: "number",
-        loads: {collection: {reconstructor: UnorderedMap.reconstruct, value: 'string'}}
+        // loads: {collection: {reconstructor: UnorderedMap.reconstruct, value: 'string'}}
+        loads: {class: UnorderedMap }
```

Comment:
Nice this case still works. I think this is worth mention as an example in readme, as one case in:
```
export class StatusDeserializeClass {
  static schema = {
```